backend/aurcache/src/api/package.rs

```diff
@@ -275,9 +275,10 @@ pub async fn get_package(
         .map_err(|e| NotFound(e.to_string()))?
         .ok_or(NotFound("id not found".to_string()))?;
 
-    let aur_info = get_info_by_name(&pkg.name)
+    let aur_info = get_package_info(&pkg.name)
         .await
-        .map_err(|e| NotFound(e.to_string()))?;
+        .map_err(|e| NotFound(e.to_string()))?
```

Comment:
Since "not found" now returns `Ok<None>` rather than an error, this here is now a legitimate error (network error most likely) that could be bubbled up as-is?

Reply:
This is a api endpoint function where we have to return a Return code with payload. So i need to map the error to the rocket Status object.

Reply:
Maybe 500 internal error then